e2e/meradm/meradm_test.go

```diff
@@ -133,6 +154,47 @@ var _ = Describe("Meradm", func() {
 			Expect(err).To(HaveOccurred())
 		})
 	})
+
+	Describe("External Updates", func() {
+		It("should run reconciler if etcd is updated elsewhere", func() {
+			meradm("service", "add", "service1", "tcp", "10.1.1.1:888", "-s=wrr", "-b=flag-1,flag-2")
+			meradm("server", "add", "service1", "172.16.1.1:555", "-w=1", "-f=masq")
+			time.Sleep(100 * time.Millisecond)
+			MerlinStderr()
+
+			// delete server directly in etcd
+			func() {
```

Comment:
We could do a little bit of refactoring and remove some of the duplication with the block immediately below.